src/ansys/units/unit_registry.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import yaml
+
+import ansys.units as ansunits
+
+
+class UnitRegistry:
+    """
+    Initializes a Unit Registry.
```

Comment:
Doc wise this one is a bit terse.  Here you would want to write down the answer to the question "What is a UnitRegistry?".